### PR TITLE
[Jar] Practice

### PR DESCRIPTION
**Pull request by Buga1chukY**

### CODE REVIEW

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (MLGMAG) [inline allNotes/JarPractice/ExtraTaskJar.md:1]: There are some best practices to write code blocks in the Markdown.
Check:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

File: allNotes/JarPractice/JarPractice.md
Comment (MLGMAG) [inline allNotes/JarPractice/JarPractice.md:1]: There is an extra file in the repo with the same title.

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (MLGMAG) [inline allNotes/JarPractice/ExtraTaskJar.md:1]: Rename file to `multipleClassFiles.md`

File: allNotes/JarPractice/JarPractice.md
Comment (MLGMAG) [inline allNotes/JarPractice/JarPractice.md:1]: Rename file to `singleClassFile.md`

File: allNotes/JarPractice/ExtraTaskJar.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	Run the `Main` method with the command: `java <file name> <first arg> <second arg>`
+
+
+4.  Print 2 input CLI arguments and the result of  `sum`  method execution.
+	Output: 
+	`First num: 5`
+    `Second num: 10`
+    `The sum: 15`
+
+5.  Create a jar file.
```
Comment (MLGMAG) [inline allNotes/JarPractice/ExtraTaskJar.md]: Different indents.
Align them

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (Buga1chukY) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: fix

File: allNotes/JarPractice/JarPractice.md
Comment (Buga1chukY) [reply allNotes/JarPractice/JarPractice.md:1]: fix

Comment (Buga1chukY) [reply allNotes/JarPractice/JarPractice.md:1]: Fix

File: allNotes/JarPractice/ExtraTaskJar.md (hunk 1 above)
Comment (Buga1chukY) [reply allNotes/JarPractice/ExtraTaskJar.md]: fix

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (Buga1chukY) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: fix

File: allNotes/JarPractice/ExtraTaskJar.md (hunk 1 above)
Comment (MLGMAG) [reply allNotes/JarPractice/ExtraTaskJar.md]: Still present

File: allNotes/JarPractice/JarPractice.md
Comment (MLGMAG) [reply allNotes/JarPractice/JarPractice.md:1]: Still present

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (MLGMAG) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: Still present

Comment (Buga1chukY) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: fix

File: allNotes/JarPractice/ExtraTaskJar.md (hunk 1 above)
Comment (Buga1chukY) [reply allNotes/JarPractice/ExtraTaskJar.md]: fix?

File: allNotes/JarPractice/JarPractice.md
Comment (Buga1chukY) [reply allNotes/JarPractice/JarPractice.md:1]: fix?

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (MLGMAG) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: Still present.
image

Comment (MLGMAG) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: image

File: allNotes/JarPractice/multipleClassFiles.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+		int num1 = Integer.parseInt(args[0]);
+		int num2 = Integer.parseInt(args[1]);
+
+		int sum = calculator.sum(num1,num2);
+	
+		System.out.println("First num:" + num1);
+		System.out.println("Second num:" + num2);	
+		System.out.println("The sum:" + sum);
+		}
```
Comment (MLGMAG) [inline allNotes/JarPractice/multipleClassFiles.md]: Format code

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (MLGMAG) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: image

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (Buga1chukY) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: fix

File: allNotes/JarPractice/multipleClassFiles.md (hunk 2 above)
Comment (Buga1chukY) [reply allNotes/JarPractice/multipleClassFiles.md]: fix

File: allNotes/JarPractice/ExtraTaskJar.md
Comment (MLGMAG) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: Still present.

Comment (Buga1chukY) [reply allNotes/JarPractice/ExtraTaskJar.md:1]: fix

File: allNotes/JarPractice/multipleClassFiles.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+		int sum = calculator.sum(num1,num2);
+
+		System.out.println("First num:" + num1);
+		System.out.println("Second num:" + num2);
+		System.out.println("The sum:" + sum);
+	}
+}
+```
+
+2.  Class  `Calculator`  has method  `sum`. Receives 2 int and return sum.
```
Comment (MLGMAG) [inline allNotes/JarPractice/multipleClassFiles.md]: integers args*

Comment (MLGMAG) [inline allNotes/JarPractice/multipleClassFiles.md]: Remove extra spaces

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: allNotes/JarPractice/multipleClassFiles.md (hunk 3 above)
Comment (Buga1chukY) [reply allNotes/JarPractice/multipleClassFiles.md]: fix

Comment (Buga1chukY) [reply allNotes/JarPractice/multipleClassFiles.md]: fix

File: allNotes/JarPractice/singleClassFile.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+````
+3. **Compile this program**
+
+	Compile the program with the command:
+ 
+        `javac <file name>`
+
+4. **Make file `MANIFEST.MF`**
+
+	Make file with the name `MANIFEST.MF` and add next content:
```
Comment (MLGMAG) [inline allNotes/JarPractice/singleClassFile.md]: Why in one file you don't use table and it this you use them?
They should be aligned.

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: allNotes/JarPractice/singleClassFile.md (hunk 4 above)
Comment (Buga1chukY) [reply allNotes/JarPractice/singleClassFile.md]: fix

**Review by MLGMAG — APPROVED** [review]

**Pull request merged**